hack/update-codegen.sh

```diff
@@ -21,12 +21,6 @@ set -o pipefail
 SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
 
-# The provider types don't need all the extra bits that generate-groups.sh
-# creates.  A simple `go generate` is enough for these.
-# Also use this for generating deepcopy for all types (so that we use the same generator).
-echo "Generating deepcopy funcs"
-go generate ./pkg/apis/...
```

Comment:
This is a part of codegen workflow, but was failing for me using the default docker comand coming from Makefile. Now it is a separate task managed by Makefile in https://github.com/openshift/machine-api-operator/pull/744/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R68